lib/reek/smells/uncommunicative_variable_name.rb

```diff
@@ -24,7 +24,7 @@ class UncommunicativeVariableName < SmellDetector
       # The name of the config field that lists the regexps of
       # smelly names to be reported.
       REJECT_KEY = 'reject'.freeze
-      DEFAULT_REJECT_SET = [/^.$/, /[0-9]$/, /[A-Z]/].freeze
+      DEFAULT_REJECT_SET = [/^[a-z]$/, /[0-9]$/].freeze
```

Comment:
This /[A-Z]/ would have never ended up here (since it would have been parsed differently) and this /^.$/ was a bug already in my mind. Should have been /[a-z]/ from the beginning.